### PR TITLE
Called the domain once it is added to decrease initial load time

### PR DESCRIPTION
**Pull request by Abhimanyu121**

### CODE REVIEW

**Comment by rekpero** [comment]
Hey @Abhimanyu121 can you run the linter/prettier and push the changes on this branch, seems like tests failed due to lint issue

File: src/components/Domain/service.ts [hunk 1]
```diff
@@ -280,6 +281,13 @@ const DomainService = {
     );
     return response.status === 200;
   },
+  callDomain(url: string) {
+    try {
+      axios.get(url);
```
Comment (rekpero) [inline src/components/Domain/service.ts:286]: I think we need to add `https://...` b/c domain is just the name without https

Comment (rekpero) [reply src/components/Domain/service.ts:286]: Tho check on the DB, if you can

File: src/components/Domain/index.ts [hunk 2]
```diff
@@ -121,6 +120,7 @@ export async function verify(
         result.domain.link
       );
     }
+    DomainService.callDomain(result.domain.name);
```
Comment (rekpero) [inline src/components/Domain/index.ts:123]: @Abhimanyu121  this should be inside the if(...) block, I think.

**Review by rekpero — CHANGES_REQUESTED** [review]
Resolve the comments

File: src/components/Domain/index.ts (hunk 2 above)
Comment (Abhimanyu121) [reply src/components/Domain/index.ts:123]: fixing

File: src/components/Domain/service.ts [hunk 3]
```diff
@@ -280,6 +281,13 @@ const DomainService = {
     );
     return response.status === 200;
   },
+  callDomain(url: string) {
+    try {
+      axios.get(`https://${url}`);
```
Comment (rekpero) [inline src/components/Domain/service.ts:286]: @Abhimanyu121 try won't work on async call.
Do something like this
```suggestion
      axios.get(`https://${url}`).then(res => console.log("CALLING DOMAIN TO GENERATE CERTS")).catch(err => console.log);
```

**Review by rekpero — CHANGES_REQUESTED** [review]
@Abhimanyu121 resolve the comment

**Comment by rekpero** [comment]
@Abhimanyu121 prettier check failed, run it once more and commit it

**Review by rekpero — APPROVED** [review]

**Pull request merged**